releasenotes/notes/fix-kinesis-newline-strip-issue-3a2419b6f29fcab8.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    botocore: Before this change, the botocore integration stripped newlines from the JSON string encoded in the data blob of Amazon Kinesis records. This change includes a terminating newline if it is present in the decoded data.
```

NIT: 
```suggestion
    botocore: Before this change, the botocore integration stripped newlines from the JSON string encoded in the data blob of Amazon Kinesis records. This change adds a terminating newline if there was one in the original decoded data.
```
This would just be a bit clearer to me, but feel free to ignore.

---

This is just a backport, so we should make the change for this in `1.x` and backport again to `1.7` 😄